sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_internal/symmetric_key.py

```diff
@@ -59,13 +66,13 @@ def __init__(self, kid=None, key_bytes=None, key_size=None):
         supported_key_wrap_algorithms = []
         key_size = len(self._key)
         if key_size >= key_size_128:
-            supported_encryption_algorithms.append(Aes128Cbc.name())
+            supported_encryption_algorithms.append(Aes128CbcPad.name())
             supported_key_wrap_algorithms.append(AesKw128.name())
         if key_size >= key_size_192:
-            supported_encryption_algorithms.append(Aes192Cbc.name())
+            supported_encryption_algorithms.append(Aes192CbcPad.name())
             supported_key_wrap_algorithms.append(AesKw192.name())
         if key_size >= key_size_256:
-            supported_encryption_algorithms.append(Aes256Cbc.name())
+            supported_encryption_algorithms.append(Aes256CbcPad.name())
             supported_encryption_algorithms.append(Aes128CbcHmacSha256.name())
```

Comment:
This makes me nervous: we don't intend to support HMAC algorithms locally but may now do so in an undocumented way.